tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html

```diff
@@ -163,10 +175,8 @@
         <div hidden$="[[!_show_host_side_chart]]">
           <div style="height:300px;" id="host_side_chart"></div>
           <p>
-            <span><b>Explanation:</b> </span>
-            <span>To improve performance, you should try to convert from </span>
-            <span>"Reading data from files on demand" (if any) to </span>
-            <span>"Reading data from files in advance". </span>
+            <span><b>What can be done to reduce above components of the host input time:</b></span>
+            <div class="recommendationDetails" id="recommendation_details"></div>
```

Comment:
We don't need this class because the element already has an ID. How about lets remove this class, and in the CSS, use the `#recommendation_details` selector.

In general, classes and IDs can both be referenced from CSS. A single web page can see a class used many times. Each ID can only be used once (be assigned to 1 element).

Reply:
Done